redash/destinations/asana.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import textwrap
+
+import requests
+
+from redash.destinations import BaseDestination, register
+
+
+class Asana(BaseDestination):
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "pat": {"type": "string", "title": "Asana Personal Access Token"},
+                "project_id": {"type": "string", "title": "Asana Project ID"},
+            },
+            "secret": ["pat"],
```

Comment:
I think we should also make the `pat` (and potentially `project_id`) required - this won't work without the PAT, and I don't know what the behavior is for not defining a project ID.

Reply:
Looking through the [docs for the endpoint used below](https://developers.asana.com/reference/createtask), it looks like we would have to set a workspace instead. So I think we should make the project ID required.